misc-utils/lsfd-unkn.c

```diff
@@ -1094,6 +1111,35 @@ static void anon_bpf_prog_free(struct unkn *unkn)
 	free(data);
 }
 
+static void anon_bpf_prog_get_more_info(struct anon_bpf_prog_data *prog_data)
+{
+	union bpf_attr attr = {
+		.prog_id = (int32_t)prog_data->id,
+		.next_id = 0,
+		.open_flags = 0,
+	};
+	struct bpf_prog_info __attribute__((aligned(sizeof(uint64_t)))) info;
```

Comment:
We already rely on the availability of `alignas()`.
That would be nicer than the attribute.

Reply:
Updated.

Reply:
Updated.